<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1375</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>571</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1375</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ingreso al sistema</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/Images/if_Turtle_204384.ico</normaloff>:/Images/if_Turtle_204384.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(57, 134, 137);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>1250</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1250</width>
      <height>600</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(167, 221, 167);
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="ledPassword">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>320</y>
       <width>380</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>380</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px; 
background-color: rgb(255, 255, 255);
border-color: rgb(147, 255, 88);</string>
     </property>
     <property name="maxLength">
      <number>16</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="psbEntrar">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>380</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton#psbEntrar {
background-color: rgb(120, 190, 151);
 }

 QPushButton#psbEntrar:hover {
background-color: rgb(56, 134, 136);
color:  rgb(255, 255, 255);
 }</string>
     </property>
     <property name="text">
      <string>   Entrar</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/Images/door_in.ico</normaloff>:/Images/door_in.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>190</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="lblUsuario">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>160</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: rgb(0, 73, 107);</string>
     </property>
     <property name="text">
      <string>Usuario</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="psbRegistrarse">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>510</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton#psbRegistrarse {
color: rgb(0, 73, 107);
background-color: rgb(120, 190, 151);
background: transparent;
 }

 QPushButton#psbRegistrarse:hover {
color:  rgb(193, 161, 96);
background: transparent;
 }</string>
     </property>
     <property name="text">
      <string>¿No tienes una cuenta? Registrate aquí</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="ledUsuario">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>210</y>
       <width>380</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>380</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px; 
background-color: rgb(255, 255, 255);
border-color: rgb(147, 255, 88);</string>
     </property>
     <property name="maxLength">
      <number>16</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblPassword">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>260</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: rgb(0, 73, 107);</string>
     </property>
     <property name="text">
      <string>Contraseña</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblTitulo">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>485</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>485</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>485</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Turtles Finder</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbImage">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>90</y>
       <width>441</width>
       <height>441</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/Images/Turtli.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>lblTitulo</zorder>
    <zorder>lbImage</zorder>
    <zorder>ledPassword</zorder>
    <zorder>psbEntrar</zorder>
    <zorder>lblUsuario</zorder>
    <zorder>psbRegistrarse</zorder>
    <zorder>ledUsuario</zorder>
    <zorder>lblPassword</zorder>
   </widget>
   <widget class="QLabel" name="lbBack">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>2</y>
      <width>1371</width>
      <height>701</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>lbBack</zorder>
   <zorder>groupBox</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>ledUsuario</tabstop>
  <tabstop>ledPassword</tabstop>
  <tabstop>psbEntrar</tabstop>
  <tabstop>psbRegistrarse</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
